src/internal/clusterstate/v2.11.0/project_metadata.go

```diff
@@ -0,0 +1,18 @@
+package v2_11_0
+
+import (
+	"context"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/migrations"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pctx"
+)
+
+func addProjectMetadata(ctx context.Context, env migrations.Env) error {
+	ctx = pctx.Child(ctx, "addProjectMetadata")
+	tx := env.Tx
+	if _, err := tx.ExecContext(ctx, `ALTER TABLE core.projects ADD COLUMN created_by TEXT NOT NULL DEFAULT ''`); err != nil {
```

Comment:
I think this is a good opportunity to add a users table under the `auth` schema and link to projects by foreign key